R/util.R

```diff
@@ -2199,6 +2199,22 @@ calculate_cohens_w <- function(chi_sq, N) {
   sqrt(chi_sq/N)
 }
 
+# Calculate Cohen's w from the following input for 2x2 AB test case.
+# - Ratio of sample size between A and B
+# - Expected overall conversion rate
+# - % difference to detect. i.e. conversion(A) - conversion(B)
```

Comment:
Update the comment. It is actually not "%" but "ratio".

Reply:
Done.